### PR TITLE
feat: [DSSC#8669qnub8] Bugs Envoyer un formulaire finaliser

### PR DESCRIPTION
**Pull request by aertyuiop12345**
Closes #

#### What has been done to verify that this works as intended?
Aucun test unitaires n'a été fait sur les nouvelles insertions de code
#### Why is this the best possible solution? Were any other approaches considered?
Non
#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
Aucun risque de regression. Les fonctionnalités impactés par ce travail concerne uniquement les boutons et la mise a jour des couleurs.
#### Do we need any specific form for testing your changes? If so, please attach one.
Aucun
#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.
Non
#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

File: collect_app/src/main/java/org/odk/collect/android/activities/InstanceUploaderListActivity.java [hunk 1]
```diff
@@ -132,31 +133,49 @@ public void onUploadButtonsClicked() {
             // items selected
             uploadSelectedFiles();
             setAllToCheckedState(listView, false);
-            toggleButtonLabel(findViewById(R.id.toggle_button), listView);
+            //toggleButtonLabel(findViewById(R.id.toggle_button), listView);
```
Comment (Baom92) [inline collect_app/src/main/java/org/odk/collect/android/activities/InstanceUploaderListActivity.java]: SUpprimer les codes commentés

**Review by Baom92 — CHANGES_REQUESTED** [review]
Le ticket a été mis à jour
Prière se référer à çà

**Review by Baom92 — APPROVED** [review]

File: strings/src/main/res/values/untranslated.xml [hunk 2]
```diff
@@ -43,6 +43,6 @@ the specific language governing permissions and limitations under the License. -
     <string name="btn_cycle" translatable="false">Mon\n Cycle </string>
     <string name="btn_about" translatable="false">About</string>
     <string name="form_selected" translatable="false">(%s)Formulaire sélectionné(s)</string>
-
+    <string name="upload_answer" translatable="false">Envoie du Formulaire</string>
```
Comment (jgnacadja) [inline strings/src/main/res/values/untranslated.xml:46]: l'envoie du formulaire n'était pas une fonction native ?

**Review by jgnacadja — COMMENTED** [review]
Need infos

File: strings/src/main/res/values/untranslated.xml (hunk 2 above)
Comment (Baom92) [reply strings/src/main/res/values/untranslated.xml:46]: C'est une fonction native. Il fallait adapter la page à la maquette. Et, pendant les tests, on a remarqué que le message affiché à l'envoi du formulaire n'est pas correcte en français

**Review by jgnacadja — APPROVED** [review]

**Review by Baom92 — APPROVED** [review]

**Review by jgnacadja — APPROVED** [review]

**Pull request merged**